### PR TITLE
fix title truncate and missing doc details tabs

### LINKED ISSUE #341: Item titles are not aligned
Package version: 1.0.0-alpha23

## Screenshots 
Screenshot 2021-01-19 at 16 47 30

Also in document details:
Screenshot 2021-01-19 at 16 49 34
the question mark should be next to the button.

### PR DESCRIPTION
**Pull request by ntarocco**
* fixes display when doc title is truncated
* fixes literature covers to be all centered
* fixes missing/wrong panels in details pages
* fixes doc titles in backoffice items/eitems
* closes #341

### CODE REVIEW

**Comment by ntarocco** [comment]
frontsite_list

frontsite_grid

frontsite_details_mobile

File: src/lib/pages/backoffice/Home/lists/OverbookedDocumentsList/state/actions.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const fetchOverbookedDocuments = () => {
 
     try {
       const response = await documentApi.list(
-        documentApi.query().overbooked().qs()
+        documentApi.query().overbooked().sortBy('-loan_requests').qs()
```
Comment (jrcastro2) [inline src/lib/pages/backoffice/Home/lists/OverbookedDocumentsList/state/actions.js:16]: Question: What does the minus sign (`-`) mean? Is it for the order?

**Review by jrcastro2 — APPROVED** [review]
LGTM! A couple of tests are failing.

File: src/lib/pages/backoffice/Home/lists/OverbookedDocumentsList/state/actions.js (hunk 1 above)
Comment (ntarocco) [reply src/lib/pages/backoffice/Home/lists/OverbookedDocumentsList/state/actions.js:16]: Yes, the trick here is that if you use React-SearchKit, then it is hiding this complexity. You will have the sort order `asc` or `desc`.
If instead you directly use Invenio APIs, which is using Elasticsearch behind, then you have have `fieldname` for ASC and `-fieldname` for DESC (unless configured in a different way).

**Review by KonstantinaStoikou — APPROVED** [review]

**Pull request merged**